elastic/apm-server
# modeldecoder/rumv3: drop standalone metricsets

## Motivation/summary

When using v3, the RUM agent only sends metricsets for breakdown metrics embedded within transactions, and never as standalone events:

https://github.com/elastic/apm-agent-rum-js/blob/52f9118e587bef1831a743eee3a3166f9eba141a/packages/rum-core/src/common/compress.js#L260

Drop the `metricsetRoot` type from modeldecoder/rumv3. Also, RUM transaction breakdown metricsets never contain tags/labels, so we can drop that field from `transactionMetricset` (nee `metricset`) type.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

Run apm-integration-testing with opeans-rum

Check that opbeans-rum breakdown metrics still show up in the APM UI, along the lines of:

![image](https://user-images.githubusercontent.com/843579/131240638-11c8f0d8-c94c-4156-a971-2d5e6d2d5d10.png)

## Related issues

None
